src/ui/content/index.tsx

```diff
@@ -13,24 +13,48 @@ interface contentProps {
 }
 
 export function Content(props: contentProps) {
-  return (
-    <Switch>
-      <Route path="/" exact>
-        <Home />
-      </Route>
-      <Route path="/login">
-        <Login />
-      </Route>
-      <Route path="/register">
-        <Register />
-      </Route>
-      <Route path="/movies">
-        <Movies />
-      </Route>
-      <Route path="/dashboard">
-        <Dashboard user={props.user} />
-      </Route>
-      <Route component={PageNotFound} />
-    </Switch>
-  );
+  if (props.user) {
+    return (
+      <Switch>
+        <Route path="/" exact>
+          <Home />
+        </Route>
+        <Route path="/login">
+          <Login />
+        </Route>
+        <Route path="/register">
+          <Register />
+        </Route>
+        <Route path="/movies">
+          <Movies />
+        </Route>
+        <Route path="/dashboard">
+          <Dashboard user={props.user} />
+        </Route>
+        <Route component={PageNotFound} />
+      </Switch>
+    );
+  }
+  else {
+    return (
+      <Switch>
+        <Route path="/" exact>
+          <Home />
+        </Route>
+        <Route path="/login">
+          <Login />
+        </Route>
+        <Route path="/register">
+          <Register />
+        </Route>
+        <Route path="/movies">
+          <Login />
```

Comment:
Now the login page will be shown under the /movies url. It can be a bit confusing and cause some issues when the user goes to the movies page after login. It is better to use a redirect the the login page in this case.